app/views/home/_dmca.html.erb

```diff
@@ -1,95 +1,177 @@
-  </p>
-  <h3 class="heading">
-    What happens to repeat copyright infringers?
-  </h3>
-  <p>
-    Pursuant to the <acronym title="Digital Millennium Copyright Act">DMCA</acronym>, we will terminate, in appropriate circumstances, users and account holders of our system and network who are repeat infringers (see 17 U.S.C. § 512 (i)(1)(A)).
-  </p>
-  <p>
-    In the event that material is removed due to a <acronym title="Digital Millennium Copyright Act">DMCA</acronym> notice, the only recourse for restoring such material is to file a counter-notice with the <acronym title="Organization for Transformative Works">OTW</acronym>. If you believe that a take-down notice which has been acted upon by the <acronym title="Organization for Transformative Works">OTW</acronym> is without legal basis, please feel free to visit the following sites as a first step in learning about filing a counter-notice:
-  </p>
+  <p>Per 17 U.S.C. § 512(f), you may be subject to liability if you knowingly and materially misrepresent your claim
+    that the content infringes on your copyright.</p>
+  <h4 class="heading" id="takedown_process">DMCA takedown process</h4>
+  <p>If the OTW receives a DMCA takedown notice that fulfills all <a href="#takedown_requirements">legal
+      requirements</a>, we will remove the content once we review the validity of the infringement claim.</p>
+  <p>When the content is removed from AO3, the information you provided in the DMCA takedown notice will be forwarded
+    to the owner of the account responsible for posting the content. We may also send a redacted version of the DMCA
+    takedown notice to <a href="https://lumendatabase.org/">Lumen</a> and other third parties at our discretion.</p>
+  <p>If the account owner believes that their content does not infringe upon your copyright, they may file a DMCA
+    counternotice disputing the takedown. The account owner can file a DMCA counternotice at any point after the
+    content has been removed; there is no time limit.</p>
+  <p>If the OTW receives a valid DMCA counternotice, we will make reasonable efforts to notify you. You will have 10
+    business days to file a lawsuit to prevent the restoration of the content. If you do not notify us within 10
+    business days that you have filed a lawsuit, we will allow the content to be restored in accordance with 17
+    U.S.C. § 512(g)(2)(B)-(C).</p>
+  <h3 class="heading" id="appeal_options">I was notified that my work was removed from AO3 due to a DMCA takedown
+    notice. What can I do?</h3>
+  <p><a href="/content#II.D.">Copyright infringement</a> and <a href="/content#II.E">plagiarism</a> are violations of
```

Comment:
```suggestion
  <p><a href="/content#II.D">Copyright infringement</a> and <a href="/content#II.E">plagiarism</a> are violations of
```
Punctuation error in the provided HTML; the live anchor does not end in a period.